main.cpp

```diff
@@ -542,13 +542,17 @@ class temperature_sensor { // Variables Implemented
         if (i2c.write(ADDR, reinterpret_cast<const char*>(buf), 1, true) == 0 &&
             i2c.read(ADDR, reinterpret_cast<char*>(buf), 1) == 0 &&
             (buf[0] & 0b11111000) == 0b11001000) {
+            isInstalled = true; //As ID was read successfully we assume that is installed
```

Comment:
Could you please put one space between the `//` and `As`?
Also, I wanted to be able to switch between `true` and `false` in build options, etc.
Ideally, it would be nice to know if it is `true` or `false` by referring to a certain port.

`//`と`As`の間に1つスペースを入れていただけますか?
また、ビルドオプションなどで`true`、`false`を切り替えられるようにしたいと思いました。
理想をいえば、あるポートを参照することで`true`か`false`かわかると良いですね。

Reply:
It might also be a good idea if the variable names are easy to understand what is being installed.

また、何がインストールされているのかわかりやすい変数名だと良いかもしれない。

Reply:
variable renamed to `successfullyDetected` with latest commit

最新のコミットで変数の名前が `successlyDetected` に変更されました

Could you please explain which different build options you mean? From my point of view the temperature sensor is either detected or not.

どのようなビルド オプションを意味するのか説明していただけますか? 私の観点からすると、温度センサーは検出されているかどうかのどちらかです。

Reply:
I was mistaken.
You were checking for a temperature sensor in the init.
If that is the case, you do not need the build option.

勘違いをしていました。
initの中で温度センサの有無を確認していたのですね。
そういうことであれば、ビルドオプションは不要です。